dali/pipeline/data/tensor.h

```diff
@@ -123,25 +122,6 @@ class Tensor : public Buffer<Backend> {
     order_.wait(order);
   }
 
-  /**
-   * @brief Loads the Tensor at index idx from the input TensorList.
-   */
-  template <typename InBackend>
-  inline void Copy(const TensorList<InBackend> &other, int idx, AccessOrder order = {}) {
```

Comment:
So what do we do now? SampleView? It doesn't have all bells and whistles that a Tensor used to have - and I'd prefer not having to haul the source TensorVector and index everywhere some tensor metadata might be needed...